webapp/src/reducers/connectedState/index.test.tsx

```diff
@@ -0,0 +1,19 @@
+import reducer, {setConnected, ConnectedState} from 'reducers/connectedState';
+
+test('should change state of connected to true on payload value equal to "true"', () => {
```

Comment:
is this make more sense?
```suggestion
test('should change the state of connected to true when payload value is "true"', () => {
```